lib/stm32wba/BLE_TransparentMode/Core/Inc/app_conf.h

```diff
@@ -444,15 +372,15 @@ typedef enum
 #define NEXT_EVENT_SCHEDULING_FROM_ISR      (1)
 
 /* Link Layer uses temperature based calibration (0 --> NO ; 1 --> YES) */
-#define USE_TEMPERATURE_BASED_RADIO_CALIBRATION  (0)
+#define USE_TEMPERATURE_BASED_RADIO_CALIBRATION  (1)
 
 #define RADIO_INTR_NUM                      RADIO_IRQn     /* 2.4GHz RADIO global interrupt */
 #define RADIO_INTR_PRIO_HIGH                (0)            /* 2.4GHz RADIO interrupt priority when radio is Active */
 #define RADIO_INTR_PRIO_LOW                 (5)            /* 2.4GHz RADIO interrupt priority when radio is Not Active - Sleep Timer Only */
 
 #if (USE_RADIO_LOW_ISR == 1)
 #define RADIO_SW_LOW_INTR_NUM               HASH_IRQn      /* Selected interrupt vector for 2.4GHz RADIO low ISR */
-#define RADIO_SW_LOW_INTR_PRIO              (14)           /* 2.4GHz RADIO low ISR priority */
+#define RADIO_SW_LOW_INTR_PRIO              (15)           /* 2.4GHz RADIO low ISR priority */
```

Comment:
I remember 14 was required to avoid an assert failure in a priority check. Have you tried to run an application to verify the behavior?

Reply:
As you said, it's better for keeping the same behavior to keep it 14 .
We need to review all the app_conf file and put some of them as build parameters.